Orange/widgets/data/tests/test_owtranspose.py

```diff
@@ -33,7 +33,30 @@ def test_parameters(self):
         self.assertListEqual(
             [a.name for a in self.get_output(self.widget.Outputs.data).domain.attributes],
             [self.zoo.domain.metas[0].to_val(m) for m in self.zoo.metas[:, 0]])
-        self.widget.feature_radio.buttons[0].click()
+        self.widget.feature_type = self.widget.GENERIC
         self.widget.apply()
         self.assertTrue(all(["Feature" in x.name for x
                              in self.get_output(self.widget.Outputs.data).domain.attributes]))
+
+    def test_transpose_custom_name(self):
+        """
+        Custom name can be set.
+        GH-2737
```

Comment:
In my opinion -- and I would want to hear what @astaric, @lanzagar and @kernc think about it -- tests should test some functionality and not specific bugs that were discovered, or be related to some specific issue in any way. Hence, I do not like comments like GH-2737. This makes our unit tests look like a bunch of tests for patches. This is irrelevant -- potential bugs must be covered by "normal" tests, but bug-specific tests.

This bothered me for some time already, but now I stumbled upon this again when I wanted to reorganize tests for this widget, and I decided to "dissolve" this test into some other test.

Again, this is just my opinion.

Reply:
but bug-specific tests. -> not bug specific tests.

Reply:
Fixing bugs or commonplace stuff like test_report, sure. But in general, it's not necessarily obvious why a certain feature was added in amendments as opposed to initially. It's customary to cross link to the issue(s) where/when the surrounding discussion occurred. Case by case; ymmv.